src/SFA.DAS.LevyTransferMatching.Domain/Extensions/PledgeStatusExtenstions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using SFA.DAS.LevyTransferMatching.Domain.Types;
+
+namespace SFA.DAS.LevyTransferMatching.Domain.Extensions
+{
+    public static class PledgeStatusExtenstions
+    {
+        public static string GetLabelForPledgeStatus(this PledgeStatus status)
```

Comment:
Pedantic, but this could just be `GetLabel` - the `ForPledgeStatus` bit is redundant, since it's a `PledgeStatus` extension method.

Reply:
Yes sure, these changes are now done, I have renamed it to GetLabel